File: spec/code_samples/Python_python3/boards@notes/patch.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

payload = "{\"body\":\"<ADD STRING VALUE>\",\"title\":\"<ADD STRING VALUE>\"}"

conn.request("PATCH", "/api/boards/notes", payload)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Php_http2/boards@notes/patch.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append('{"body":"<ADD STRING VALUE>","title":"<ADD STRING VALUE>"}');

$request->setRequestUrl('https://www.directfreight.com/api/boards/notes');
$request->setRequestMethod('PATCH');
$request->setBody($body);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();